src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,42 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_AGE = 18;
+    private static final int MIN_LOGIN_LENGTH = 5;
+    private static final int MIN_PASSWORD_LENGTH = 5;
```

Comment:
The minimum password length constant is set to 5, but according to the checklist, it should be 6. Please correct the constant value to align with the task requirements.